USAGE.md

```diff
@@ -83,9 +83,112 @@ Simplest query:
 SELECT * FROM logs
 ```
 
-Also Elasticsearch queries are supported, you can write them in `QUERY` function:
+Number of columns in logs table is dynamic and can ago into few hundreds, so it is better to ask
+for specific columns.
+
+### Log Table
+
+The only supported table at the moment is `log` table. There might be more tables in the future.
+
+You can navigate objects like `coralogix.metadata.subsystemName`.
+
+```
+SELECT text, coralogix.metadata.subsystemName FROM logs
+```
+
+
+<!-- FIXME I would like to add standard columns, what are they and also metion nonstandard one
```

Comment:
Example of the standard `coralogix` object:
```json
"coralogix": {
                        "branchId": "71fb8364-75d5-3cfc-b1e5-b8fd4386892d",
                        "jsonUuid": "35231f56-b46a-3806-c6b7-09f549235ba8",
                        "logId": "c5b8f92c-1586-4656-9fbb-ba07a1ca749c",
                        "metadata": {
                            "applicationName": "default",
                            "category": "CORALOGIX",
                            "className": "",
                            "companyId": 748,
                            "computerName": "fluentd-659d7c685f-j2hwc",
                            "ipAddress": "",
                            "methodName": "",
                            "processName": null,
                            "sdkId": null,
                            "severity": 1,
                            "subsystemName": "bulks-ingester",
                            "threadId": 47181310606400
                        },
                        "templateId": "3c12a35e-1794-11eb-8729-de34acb4e34f",
                        "timestamp": "2021-01-27T02:12:45.401"
                    }
```

Reply:
Added it to docs

Reply:
You can take a look again @iravid